samples/features/sql-management-objects/prep/restore.sh

```diff
@@ -0,0 +1,4 @@
+sleep 35s
```

Comment:
I'd suggest adding detailed comments of what these scripts are doing since this is going to be run by people who may not be the most familiar with these tools. And just in general more information in customer-facing docs is good - for example I'm curious myself what this sleep is for